frontend/packages/kubevirt-plugin/src/statuses/v2v/v2vvmware-status.ts

```diff
@@ -14,6 +15,13 @@ const hasV2vVMWareStatus = (providerCR: K8sResourceKind, flags: V2VProviderStatu
 
   const status = V2VProviderStatus.fromPhase(getStatusPhase(providerCR));
 
+  if (
+    providerCR?.kind === OVirtProviderModel.kind &&
```

Comment:
IMO it would be preferred to use this in a place where we render it. Not here where we resolve the status

Reply:
Done!